deploy/docker/sysrepo-netopeer2/Dockerfile

```diff
@@ -79,22 +88,17 @@ RUN \
       cd /opt/dev && \
       git clone https://github.com/sysrepo/sysrepo.git && cd sysrepo && \
       mkdir build && cd build && \
-      cmake -DENABLE_TESTS=OFF -DCMAKE_BUILD_TYPE="Release" -DCMAKE_INSTALL_PREFIX:PATH=/usr -DREPOSITORY_LOC:PATH=/etc/sysrepo .. && \
+      cmake -DENABLE_TESTS=OFF -DCMAKE_BUILD_TYPE="Release" -DCMAKE_INSTALL_PREFIX:PATH=/usr -DREPO_PATH:PATH=/etc/sysrepo .. && \
```

Comment:
For `Release` build target, this `REPO_PATH` should be default so it is redundant to set it explicitly.